content/dev/dev-guide.md

```diff
@@ -0,0 +1,325 @@
+   - The `sys.kelvin` file specifies the kernel version it's compatible with.
+   - The optional `desk.ship` file specifies the original publisher of this desk. Since we merged this from `%garden` its currently set to `~mister-dister-dozzod-dozzod`. Let's change this to our fake ship `~zod`, in practice you'll set this to the ID you want to be seen as the publisher (likely your urbit ID). Currently there is no verification that makes sure publishers are honest about this, but eventually there will be.
+   - ```bash
+     echo "~zod" > ~/urbit-dev/zod/hello/desk.ship
+     ```
+ - If we had Gall agents in this desk which should be automatically started when the desk is installed, we'd add them to a hoon list in the `desk.bill` file. From our `%garden` merge we currently have this:
+   - ```bash
+      :~  %docket
+          %treaty
+          %hark-store
+          %hark-system-hook
+          %settings-store
+      ==
+     ```
+ - Since we're not adding any agents in this example, we'll remove the `desk.bill` file and some other garden specific directories.
+   - ```bash
+       rm ~/urbit-dev/zod/hello/desk.bill app gen ted
+     ```
+ - If you `ls` you should see the following in `hello`:
+   - `desk.docket-0 desk.ship     lib           mar           sur           sys.kelvin`
+ - The final file we need to edit is `desk.docket-0`. This one's more complicated, so we'll need a text editor.
+   - ```bash
+     nano desk.docket-0
+     ```
+   - In the text editor you'll see the following that came in from `%garden`:
+   - ```bash
+        :~ title+'System'
+           info+'An app launcher for Urbit.'
+           color+0xee.5432
+           version+[1 0 2]
+           website+'https://tlon.io'
+           license+'MIT'
+           base+'grid'
+           glob-http+['https://bootstrap.urbit.org/glob-0v4.64ana.19ug9.ik7l6.og080.68ce4.glob' 0v4.64ana.19ug9.ik7l6.og080.68ce4]
+        ==
+     ```
+   - We're going to modify this for our app to the below:
+   - ```bash
+     :~  title+'Hello'
+         info+'A simple hello world app.'
+         color+0x81.88c9
+         version+[0 0 1]
+         website+'https://urbit.org/docs/userspace/dist/guide'
+         license+'MIT'
+         base+'hello'
+         glob-ames+[~zod 0v0]
+         image+'https://media.urbit.org/docs/userspace/dist/wut.svg'
+     ==
+     ```
+  - This is a `docket` file. This file sets various options for the desk with a tile and usually a brower-based front-end of some kind. 
+  - It mainly configures the appearance of an app's tile, the source of its glob (bundle of client resources like HTML, JS, and CSS files), and some additional metadata.
+    - We've given the app a `%title` of "Hello", which will be displayed on the app tile and will be the name of the app when others browse to install it. 
+    - We've given the app tile a `%color` of #8188C9, and specified the URL of an `%image` to display on the tile.
```

Comment:
Honestly might be better to just say "given the app a title of "Hello"" instead, same for the others. The reader may not know about `a+b` sugar and just assume all names are `%names` or w/e. It's overkill.

Also we mention "clause" in a couple places but never explain what that means.